src/OpenLoco/src/Windows/CompanyList.cpp

```diff
@@ -27,34 +27,43 @@ using namespace OpenLoco::Literals;
 
 namespace OpenLoco::Ui::Windows::CompanyList
 {
-    static loco_global<Colour[32], 0x004F9442> _cargoLineColour;
     static loco_global<currency32_t[32][60], 0x009C68F8> _deliveredCargoPayment;
     static loco_global<uint16_t, 0x009C68C7> _word_9C68C7;
-    static loco_global<uint16_t, 0x0113DC7A> _graphLeft;
-    static loco_global<uint16_t, 0x0113DC7C> _graphTop;
-    static loco_global<uint16_t, 0x0113DC7E> _graphRight;
-    static loco_global<uint16_t, 0x0113DC80> _graphBottom;
-    static loco_global<uint16_t, 0x0113DC82> _graphYOffset;
-    static loco_global<uint16_t, 0x0113DC84> _graphXOffset;
-    static loco_global<uint32_t, 0x0113DC86> _graphYAxisLabelIncrement;
-    static loco_global<uint16_t, 0x0113DC8A> _graphLineCount;
-    static loco_global<uint32_t[32], 0x0113DC8C> _graphYData;
-    static loco_global<uint32_t, 0x0113DD0C> _graphDataTypeSize;
-    static loco_global<uint16_t[32], 0x0113DD10> _graphDataStart;
-    static loco_global<uint32_t, 0x0113DD50> _dword_113DD50;
-    static loco_global<PaletteIndex_t[32], 0x0113DD54> _graphLineColour;
-    static loco_global<uint16_t, 0x0113DD74> _graphDataEnd;
-    static loco_global<uint16_t, 0x0113DD76> _graphXLabel;
-    static loco_global<uint32_t, 0x0113DD78> _graphXAxisRange;
-    static loco_global<uint32_t, 0x0113DD7C> _dword_113DD7C;
-    static loco_global<uint16_t, 0x0113DD80> _word_113DD80; // graphXAxisIncrement?
-    static loco_global<uint16_t, 0x0113DD82> _graphXAxisLabelIncrement;
-    static loco_global<uint16_t, 0x0113DD84> _graphYLabel;
-    static loco_global<uint32_t, 0x0113DD86> _dword_113DD86;
-    static loco_global<uint32_t, 0x0113DD8A> _dword_113DD8A;
-    static loco_global<uint32_t, 0x0113DD8E> _dword_113DD8E;
-    static loco_global<uint8_t, 0x0113DD99> _byte_113DD99;
-    static loco_global<uint16_t[32], 0x0113DD9A> _graphItemId;
+
+#pragma pack(push, 1)
+    struct GraphSettings
```

Comment:
We should split graph parameters and graph data after we implemented the drawing, this bundles too much in one struct. For now we can leave it be.

Reply:
Agreed. I'd also like to move it to e.g. Graph.h and put the plotting code in an accompanying unit file, e.g. Graph.cpp.